starter-code/index.html

```diff
@@ -3,40 +3,81 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+    <nav>
+      
+  <img class="spotifylogo" src="./images/spotify-logo.png" alt="spotifylogo">
+
+  <ul class="top-right">
+<li><a href="#">Premium</a></li>
+<li><a href="#">Discover</a> </li>
+<li><a href="#">Help</a> </li>
+<li><a href="#">Download</a></li>
+</ul>
+
+</nav>
+<div class="container">
+  <div class="containertitle">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer.</p>
+<p>Listen to the right music, wherever you are.</p>
+</div>
+</div>
+
+<div class="container2">
+  <div class="container2head">
+<h2>What’s on Spotify?</h2>
+<hr>
+</div>
+</div>
+<div class="container3">
```

Comment:
Please start getting into the habit of properly spacing/tabbing ALL of your divs, because all of your divs are on the same left side as the body tag, so now it is hard to follow where starting and closing tags start and end.